packages/lib/services/e2ee/cryptoShared.ts

```diff
@@ -36,12 +36,11 @@ export const generateNonce = async (nonce: Uint8Array) => {
 };
 
 export const increaseNonce = async (nonce: Uint8Array) => {
-	const carry = 1;
 	const end = nonce.length - nonceCounterLength;
 	let i = nonce.length;
 	while (i-- > end) {
-		nonce[i] += carry;
-		if (nonce[i] !== 0 || carry !== 1) {
+		nonce[i] += 1;
+		if (nonce[i] !== 0) {
 			break;
```

Comment:
Could this affect data that has already been encrypted with the new encryption method? From the name I'm guessing that not but please confirm. How is this function used?

Reply:
This doesn't affect the encrypted data at all. The old implementation and and the new implementation generate exactly the same output with same input (nonce in this case). It's just a optimization.
This function is used when a new chunk of data is encrypted. Each trunk of data is encrypted with the same key and a different nonce, and this function is used to change the nonce.

Reply:
Thanks for confirming!